adapters/tappx/tappx.go

```diff
@@ -143,7 +148,7 @@ func (a *TappxAdapter) buildEndpointURL(params *openrtb_ext.ExtImpTappx, test in
 		thisURI = thisURI + "&ts=" + strconv.Itoa(int(t))
```

Comment:
We noticed this when writing the tests. Since the parameter added to `thisURI` is a timestamp we cannot predict its value and we don't know how to include this in a written test case. Could you give us some help?

Reply:
The `strconv.Itoa(int(t))` in line 147 seems to return a 13 digit number
```
                //Some random code I wrote to watch the contents of strconv.Itoa(int(t)) with the debugger
   151:         stringWithTime := "=" + strconv.Itoa(int(t))
=> 152:         if stringWithTime == "" {
   153:                 fmt.Printf("ok \n")
   154:         }
(dlv) p stringWithTime
"=1571940732071"
//1234567890123
```
To create the test case withouth knowing the exact digits that `strconv.Itoa(int(t))` maybe we can assert with a regex match:
```
func TestNewTestToAssertEndpointURL() {    

    // build endpoint URL here

    assert.True(t,regexp.MatchString(`&ts=[0-9]\{13\}`,endpointURL)
}
```
Or maybe we could use the `time` library's `Parse` function [https://golang.org/pkg/time/#Parse|https://golang.org/pkg/time/#Parse] to turn the string unto a valid `Time`object and `assert.NoError` to make sure the test passes
```
func TestNewTestToAssertEndpointURL() {    

    // build endpoint URL here

    lastDigitsOfEndpoint := //String function to strip the last n characters from the endpoint

    _, err := Parse(layout, lastDigitsOfEndpoint)

    assert.NoError(t,err,"Error: `lastDigitsOfEndpoint` could not be parsed to a valid Time object")
}
```